### DISCUSSION TITLE
M154 Auto Report Real Position Feature

### DISCUSSION THREAD

**Discussion by sarvenn** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2784] [Q&A]
Hi, a new feature has been added to Marlin. M154 Auto report position
Should we enable it or does TFT firmware requires an update for it. Would there be a benefit for TFT users?
AUTO_REPORT_REAL_POSITION
https://github.com/MarlinFirmware/Marlin/pull/25738
![image](https://user-images.githubusercontent.com/66178082/235068799-15ee69ba-8dee-430d-8451-1188d3a38873.png)

**Comment by sarvenn** [comment]
@kisslorand , @digant73 , @guruathwal , @bigtreetech

**Comment by kisslorand** [comment]
The TFT doesn't know if the received coordinates are real or projected. However the TFT polls/receives the coordinates only periodically so whichever coordinates it receives (real or projected) it doesn't matter. As far as I know the TFT needs the coordinates only when the motors are stationary and in that case the projected coordinate equals to the real one. I do not know any situation where the TFT needs the coordinates between a start and an end point. As for the user to see the real coordinates, it is not possible because of the periodical update of them so it doesn't matter that at the time of reading the coordinates they are real or projected. 
I might be utterly wrong but anyhow these are my 2 cents on the matter.
